### PR TITLE
TPC-DS should be able to run by any given username

### PR DESCRIPTION
**Pull request by chandankunal**
Authored-by: Chandan Kunal chandank1@vmware.com

### CODE REVIEW

File: tpcds.sh [hunk 1]
```diff
@@ -16,9 +16,6 @@ export TPC_DS_DIR
 
 # Check that pertinent variables are set in the variable file.
 check_variables
-# Make sure this is being run as gpadmin
-check_admin_user
-# Output admin user and multi-user count to standard out
```
Comment (ajs95) [inline tpcds.sh:21]: This comment shouldn't be deleted

**Comment by chandankunal** [comment]
Test location: root@10.202.89.133
For gpadmin user: /home/gpadmin/TPC-DS
       log: gpadmin-log

For other(here chandan) user: /home/chandan/TPC-DS
      log: chandan-log

**Review by ajs95 — APPROVED** [review]

File: tpcds_variables.sh [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # shellcheck disable=SC2148
 # environment options
-export ADMIN_USER="gpadmin"
+# user should have equivalent access privilege of gpadmin. eg.
+# 1. Access privilege on gpdata directory and sub-directory on each node.
+# 2. Should have read access to log folder across nodes.
+# 3. Access privilege similar of gpadmin on databases.
+# 4. Proper authentications set up in pg_hba.conf
+
+USER=$(whoami)
```
Comment (deenav) [inline tpcds_variables.sh:9]: Keep USER variable as ADMIN_USER since it requires an additional access than the normal user account.

**Review by deenav — COMMENTED** [review]
Update the Pre-requisites section in README.md file as non-gpadmin user could be able to run the tpcds and the access privileges required for that user.

File: 03_ddl/rollout.sh [hunk 3]
```diff
@@ -8,6 +8,7 @@ init_log "${step}"
 get_version
 
 filter="gpdb"
+user=$(whoami)
```
Comment (deenav) [inline 03_ddl/rollout.sh:11]: Try to reuse the $USER from tpcds_variables.sh file

File: tpcds_variables.sh (hunk 2 above)
Comment (chandankunal) [reply tpcds_variables.sh]: Done.

File: 03_ddl/rollout.sh (hunk 3 above)
Comment (chandankunal) [reply 03_ddl/rollout.sh]: Done.

**Review by deenav — APPROVED** [review]

**Pull request merged**